src/index.js

```diff
@@ -1,60 +1,103 @@
+import {loadPeople, loadPlanets, loadFilms, loadSpecies, loadVehicles} from './loadData.js';
+
+let category;
+let categorySelected = false;
+let currentPageInfo;
+
 const cardsField = document.querySelector('.card-body');
 const categories = document.querySelectorAll(`.nav-link`);
 const paginator = document.createElement('ul');
-const loader = document.querySelector('.starwars-demo')
-let category;
+const introImage = document.querySelector('.starwars-demo');
+const loader = document.querySelector('.lds-facebook');
+const search = document.getElementById('search');
+const searchField = document.querySelector('.search-field');
+const save = document.getElementById('save');
+const load = document.getElementById('load');
+const themeSwitcher = document.getElementById('customSwitch1');
+
+
+window.onload = function loadSavedData() {
+    let theme = localStorage.getItem('theme');
+
+    if (theme === 'dark') {
+        themeSwitcher.checked = true;
+    } else {
+        themeSwitcher.checked = false;
+    }
+    changeTheme();
+
+}
 
 categories.forEach(el => {
     el.addEventListener('click', (e) => {
         categories.forEach(el => {
             if(el.classList[2]){
```

Comment:
Don't use magic numbers. Create variable with clear name about what it is

Reply:
Solve it by adding variable.

Reply:
Yep, that's neat.